templates/web/base/report/_main.html

```diff
@@ -60,6 +60,21 @@ <h1 class="moderate-display">[% problem.title | html %]</h1>
         [% INCLUDE 'report/_report_meta_info.html' %]
     </p>
 
+  [% IF c.user_exists AND c.user.id == problem.user_id AND !problem.anonymous %]
+    <!-- TODO: Do we want to ajax this in? -->
+    <!-- TODO: Should this work without javascript too? -->
```

Comment:
Yes to the latter :) I would put this in a separate template file (perhaps merged with the update one, it's similar enough) so that non-JS can have a link to a separate page that just shows that template.